src/core/Curl/Handler.php

```diff
@@ -811,6 +828,14 @@ private function execute()
             }
         }
 
+        if ($this->cookieJar && $this->cookieJar !== '') {
```

Comment:
According to [the doc](https://curl.se/libcurl/c/CURLOPT_COOKIEJAR.html), `Specify "-" as filename to instead have the cookies written to stdout.`.

Maybe we can add a TODO and temporarily fail it directly.

Reply:
done,